bika/lims/browser/analyses/view.py

```diff
@@ -1069,7 +1069,8 @@ def _folder_item_report_visibility(self, analysis_brain, item):
         full_obj = self.get_object(analysis_brain)
         item['Hidden'] = full_obj.getHidden()
         if IRoutineAnalysis.providedBy(full_obj):
-            item['allow_edit'].append('Hidden')
+            if self.has_permission('Modify portal content'):
```

Comment:
That was actually by purpose, because how else can we allow Lab Managers or Publishers to set the Hidden Attribute for Analyses?

Reply:
Understand. Then, I would suggest to add a specific permission for this field in `analysis_workflow` (say "Field: Edit Hidden") and check this permission for analysis here. Even if Labman and others do not have the permission "Modify portal content" for this analysis, they can still have the permission "Field: Edit Hidden" granted. Since you bypass "Modify portal content" permission in the ajax end-point, it would work and at least, can give us more fine-grained control.